### PR TITLE
Blog app - creating data model

### PR DESCRIPTION
**Pull request by fed1k**
* Create a new app. Give it a meaningful name.
* Initialize your project with Git.
* Make sure that your project has Postgres database set up.
* Build your project schema.
  * Create and run the necessary migration files.
  * Table and column names should match the ERD diagram.
    * note: photo for users table should be a link to an image
  * Foreign keys should be included.
  * All columns that are foreign keys should have a corresponding index.

### CODE REVIEW

File: db/migrate/20220705210803_create_likes.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :users, null: false, foreign_key: true
```
Comment (elisha2kyakpo1) [inline db/migrate/20220705210803_create_likes.rb]: - [ ] Let's reference author, this way we can achieve the `author_id` shown on the ERD diagram.

File: db/migrate/20220705211412_create_comments.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :users, null: false, foreign_key: true
```
Comment (elisha2kyakpo1) [inline db/migrate/20220705211412_create_comments.rb]: - [ ] Let's reference author, this way we can achieve the `author_id` shown on the ERD diagram. If we look at the ERD we can't see `user_id` as foreign_key but `author_id`

File: db/migrate/20220705211831_create_posts.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.references :users, null: false, foreign_key: true
```
Comment (elisha2kyakpo1) [inline db/migrate/20220705211831_create_posts.rb]: - [ ] Let's reference author, this way we can achieve the `author_id` shown on the ERD diagram. If we look at the ERD we can't see `user_id` as foreign_key but `author_id`

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @fed1k, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- The PR title and description looks professional 👍
- No linter errors 👍
- Descriptive commit messages 👍

## Required Changes ♻️
_Check the comments under the review._
- [ ] Kindly add a descriptive readme file.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Let's make sure we follow instructions in each milestone, this milestone is about data models, another milestone for controllers and views will come, and again, we are not allowed to use scaffold generator for creating the project files.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20220705210803_create_likes.rb (hunk 1 above)
Comment (fed1k) [reply db/migrate/20220705210803_create_likes.rb]: @elisha2kyakpo1 Hello Mr.
How can I  reference `author_id` with this command `rails generate migration Post user:references` ?

Comment (elisha2kyakpo1) [reply db/migrate/20220705210803_create_likes.rb]: @fed1k, search about having custom foreign keys in Rails, this means that even at model level we should make some changes.

Comment (elisha2kyakpo1) [reply db/migrate/20220705210803_create_likes.rb]: @fed1k, try explicitly adding this line of code `t.references :author, null: false, foreign_key: {to_table: :users}`, but again like I said, you should make some changes at model level associations as well.

Comment (fed1k) [reply db/migrate/20220705210803_create_likes.rb]: Thanks for the help @elisha2kyakpo1

**Review by zainsadaqat — APPROVED** [review]
# APPROVED!!!! 🎉 🎉 🎉 🎉 🎉 🎉
## Hey @fed1k  😊 😃 

#### Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**